src/util/Util.js

```diff
@@ -200,12 +200,12 @@ class Util extends null {
    */
   static escapeItalic(text) {
     let i = 0;
-    text = text.replace(/(?<=^|[^*])\*([^*]|\*\*|$)/g, (_, match) => {
+    text = text.replaceAll(/(?<=^|[^*])\*([^*]|\*\*|$)/g, (_, match) => {
```

Comment:
Unnecessary, please revert to `replace` as RegExp with global flag will go thru all instances.
```suggestion
    text = text.replace(/(?<=^|[^*])\*([^*]|\*\*|$)/g, (_, match) => {
```